pandas/tests/plotting/test_datetimelike.py

```diff
@@ -406,11 +406,9 @@ def test_get_finder(self):
     def test_finder_daily(self):
         day_lst = [10, 40, 252, 400, 950, 2750, 10000]
 
-        if (self.mpl_ge_3_0_0 or not self.mpl_ge_2_0_1
-                or (self.mpl_ge_2_1_0 and not self.mpl_ge_2_2_2)):
-            # 2.0.0, 2.2.0 (exactly) or >= 3.0.0
+        if self.mpl_ge_3_0_0 or self.mpl_eq_2_2_2:
```

Comment:
see my above comment, you should just be able to do : ``not self.mpl_ge_3_0_3`` to get this condition